src/utils/eip712/defaults.ts

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable no-dupe-class-members */
+/* eslint-disable no-unused-vars */
```

Comment:
these still needed?

Reply:
didnt need no-unused-vars, and i was able to reduce the scope of no-dupe-class-members, pushing a commit for that now 👍